packages/components/src/modal/style.scss

```
@@ -92,6 +92,7 @@
 	h1 {
 		line-height: 1;
 		margin: 0;
+		color: $gray-900;
```

```suggestion
		color: $components-color-foreground;
```
In the components package this is the appropriate variable to apply for `color` (so that it can be themed).